python/src/shims.py

```diff
@@ -0,0 +1,19 @@
+"""
+openturns<=1.13 compatibility shims
+"""
+
+import openturns as ot
+from openturns import *
```

Comment:
why the second import ?

Reply:
This one is to import all symbols from.

We first import all symbols, then overload our distribution ctors.

So at the end we are able to use all symbols from openturns.shims and not only our ctors:


```
import openturns.shims as ot
dist1 = ot.Arcsine()
dist2 = ot.Beta(r,t,...)
```

The only change in user scrips is importing "openturns.shims", and not:
```
import openturns as ot
import openturns.shims
dist1 = ot.Arcsine()
dist2 = openturns.shims.Beta(r, t, a, b)
```

Reply:
Ahhh I start to see what you have in mind

Reply:
`dist2` docstring refers to the new implementation docstring. Isn't it ? How to provide the correct information ? 
Btw, warning messages should be added for each of the distributions

Reply:
That's not possible. I'm afraid.
No need to add warnings, the user used openturns.shims on purpose.